featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -169,18 +169,20 @@ def test_anon_s3_csv(es):
     assert es.__eq__(new_es, deep=True)
 
 
-def tests_s3_profile_deserialize(es):
-    error_text = "The config profile (.*) could not be found"
-    with pytest.raises(ProfileNotFound, match=error_text):
-        deserialize.read_entityset(S3_URL, profile_name=TEST_CONFIG)
+def tests_s3_check_profile(es):
+    session = boto3.Session()
+    assert session.get_credentials().access_key is not TEST_KEY
 
 
 @pytest.fixture
 def s3_client():
+    _environ = dict(os.environ)
```

Comment:
Need to copy and reset environment variables as moto sets aws credentials to test credentials and thus will fail consecutive tests.